layout.go

```diff
@@ -0,0 +1,39 @@
+package irutil
+
+import (
+	"fmt"
+
+	"github.com/llir/llvm/ir/types"
+)
+
+// Layout provides configuration about metadata of compiler
```

Comment:
Documentation from https://releases.llvm.org/12.0.0/docs/LangRef.html#data-layout

```suggestion
// Layout specifies how data is to be laid out in memory.
```